src/server/Document/index.jsx

```diff
@@ -34,14 +36,19 @@ const renderDocument = async ({
           routes={routes}
           data={data}
           bbcOrigin={bbcOrigin}
-          context={{}}
+          context={context}
           service={service}
           isAmp={isAmp}
         />,
       ),
     ),
   );
 
+  if (context.url) {
+    // https://alligator.io/react/react-router-ssr/
```

Comment:
That's a big article. You should quote with the only relevant sentence from that article 

> React Router automatically adds an url property with the redirected url to the context object when a Redirect component is used - https://alligator.io/react/react-router-ssr/